src/types/mod.rs

```diff
@@ -50,24 +60,43 @@ pub enum BlockStatus {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct BlockDetails {
+    /// Number of the block.
     pub number: u64,
+    /// Corresponding L1 batch number.
     pub l1_batch_number: u64,
+    /// Address of the operator who committed the block.
     pub operator_address: Address,
+    /// Version of the ZKsync protocol the block was committed under.
     pub protocol_version: Option<String>,
+    /// Unix timestamp when the block was committed.
```

Comment:
Are you sure? I would assume that it's the timestamp of when the first transaction in the block was processed (e.g. when it was opened), but not 100% sure.

Reply:
You are right, I've checked with Stas. Fixed the annotation.